content/src/main/content/jcr_root/apps/acs-commons/authoring/audio-asset-finder/.content.xml

```diff
@@ -1,6 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          jcr:primaryType="cq:ClientLibraryFolder"
+          jsProcessor="[default:none,min:gcc]"
+          allowProxy="{Boolean}true"
+          categories="[acs-commons.authoring.audio-asset-finder]"
+          dependencies="[cq.authoring.editor]"/>
+<!--
+TODO Is this OK?
```

Comment:
@justinedelson  does this look ok to you? i don't recall off the top of my head how the "hook" clientlibs work.

Reply:
@davidjgonzalez so the idea is that a "consumer" of ACS AEM Commons would create a "proxy" client library at the location of their choosing with the category 'cq.authoring.edit.hook.assetfinder` and embed `acs-commons.authoring.audio-asset-finder`?

If so, yes, that should work.